### PR TITLE
rm black

### PR DESCRIPTION
**Pull request by AxelGard**

### CODE REVIEW

**Review by DennisBerntsson — COMMENTED** [review]
test comment for functionality.

File: requirements.txt [hunk 1]
```diff
@@ -1,10 +1,8 @@
 appdirs==1.4.4
 astroid==2.5.1
 attrs==20.3.0
-black==20.8b1
```
Comment (AxelGard) [inline requirements.txt:4]: test

File: requirements.txt [hunk 2]
```diff
@@ -1,10 +1,8 @@
 appdirs==1.4.4
 astroid==2.5.1
 attrs==20.3.0
```
Comment (aannalarsson) [inline requirements.txt:3]: hej

File: requirements.txt [hunk 3]
```diff
@@ -1,10 +1,8 @@
...
-black==20.8b1
 click==7.1.2
 Cython==0.29.21
-flake8==3.8.4
 iniconfig==1.1.1
```
Comment (AxelGard) [inline requirements.txt:6]: :1st_place_medal:

File: requirements.txt (hunk 1 above)
Comment (aannalarsson) [reply requirements.txt:4]: Bra!

**Review by aannalarsson — APPROVED** [review]

**Comment by AxelGard** [comment]
```python
def add():
 pass
```

**Pull request merged**